test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -245,6 +245,33 @@ Describe "ConsoleHost unit tests" -tags "Feature" {
         }
     }
 
+    Context "-LoadProfile Commandline switch" {
+        BeforeAll {
+            if (Test-Path $profile) {
```

Comment:
It seems `$profile` is null on CIs.
Also can we change the file on containers or it is read-only?